docs/refget-design.md

```diff
@@ -0,0 +1,83 @@
+# Design notes: Modos x Refget
+
+
+## Goal
+Use a [refget](http://samtools.github.io/hts-specs/refget.html) api in modos to handle reference sequences in an unambigous and standardized way.
+
+## Considerations
+1. GA4GH published a [Refget Sequence Collections](https://ga4gh.github.io/refget/seqcol/) specification (v1 approved 2025) to:
+    *   allow retrival of sequence collections (usually __reference
+ genomes__)
+    *   specify __sequence collection hashes__, that are composed from the __sequence hashes__ and __sequence attributes__ (names, length,?)(see [Seqcol standard](#seqcol-standard))
+    *   assess compatability of sequence collections 
+        *   includes coordinate systems based on sequence attributes
+2. A pangenomes standard is planned
+3. CRAM uses reference sequence ids (not sequence collections)
+
+### The Refget umbrella
+![refget-umbrella](https://ga4gh.github.io/refget/img/refget-umbrella.svg)
+
+### Seqcol standard
+![seqcol_abstract_simple](https://ga4gh.github.io/refget/img/seqcol_abstract_simple.svg)
+
+## Existing tools
+1. [refget-cloud](https://github.com/ga4gh/refget-cloud): Python-based Refget v1 web service compatible with s3 storage
+    - :heavy_check_mark: small and simple tool
+    - :heavy_check_mark: part of the official ga4gh github organisation
+    - :heavy_minus_sign: not sure how much this is maintained (last commit 5 years ago)
+    - :heavy_minus_sign: associated tooling in perl ([ena-processor](https://github.com/andrewyatz/ena-refget-processor))
```

Comment:
maybe we could reimplement that? from the [algorithm outline](https://github.com/andrewyatz/ena-refget-processor/blob/7242d9c1c4cf4159d04dd6b4ca48b55977992669/docs/algorithm_outline.md?plain=1#L3), it seems that it's downloading genbank+ csv and extracting to json (and fasta?)